driver/src/models.rs

```diff
@@ -55,26 +22,21 @@ pub struct State {
   pub balances: Vec<u128>,
 }
 
-impl State {
+impl RollingHashable for State {
   //Todo: Exchange sha with pederson hash
-  pub fn hash(&self) -> Result<[u8; 32], error::DriverError> {
-    let mut hash: [u8; 32] = [0; 32];
+  fn rolling_hash(&self) -> H256 {
+    let mut hash = vec![0u8; 32];
     for i in &self.balances {
-      let mut bs = [0u8; 64];
-      bs.as_mut()
-        .write_u128::<LittleEndian>(*i)?;
-      for i in 0..32 {
-        bs[i + 32] = bs[i];
-        bs[i] = hash[i];
-      }
-      let bytes: Vec<u8> = bs.to_vec();
+      let mut bs = [0u8; 32];
+      bs.as_mut().write_u128::<LittleEndian>(*i).unwrap();
```

Comment:
Why do we unwrap instead of propagating the error?

Reply:
I think it's fine to panic in this case, because we know that this should never happen. We put i which is a balance (u128) into a 32 byte buffer, which should always work. 
If not, there is nothing our app can do and it should go up in flames.